cmd/skaffold/app/cmd/fix.go

```diff
@@ -66,10 +66,10 @@ func doFix(_ context.Context, out io.Writer) error {
 	} else if overwrite {
 		toFile = opts.ConfigurationFile
 	}
-	return fix(out, opts.ConfigurationFile, toFile, toVersion)
+	return fix(out, opts.ConfigurationFile, toFile, toVersion, overwrite)
 }
 
-func fix(out io.Writer, configFile, outFile string, toVersion string) error {
+func fix(out io.Writer, configFile, outFile string, toVersion string, overwrite bool) error {
```

Comment:
why do you need to pass this as a bool? isn't it already accessible / being used as a flag?

Reply:
It wasn't accessible. We are calling fix method directly which I believe previously didn't have a way of setting this flag.